__tests__/app.test.js

```diff
@@ -62,7 +62,46 @@ describe("GET /api/articles/:article_id", () => {
         );
       });
   });
+  test("Status: 200, returns a specific object based on input", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.article_id).toEqual(1);
+        expect(body.articles.title).toEqual(
```

Comment:
- You can toEqual then input the values from the object you expect. 
e.g. toEqual({
title: "the title you want",
article_id: 1, 
votes: 100
})
etc.